packages/server/postgres/queries/updateTeamByTeamId.ts

```diff
@@ -1,19 +1,21 @@
 import catchAndLog from '../../postgres/utils/catchAndLog'
 import {
   IUpdateTeamByTeamIdQueryParams,
+  IUpdateTeamByTeamIdQueryResult,
   updateTeamByTeamIdQuery
 } from '../../postgres/queries/generated/updateTeamByTeamIdQuery'
 import getPg from '../../postgres/getPg'
 
 const updateTeamByTeamId = async (
   teamUpdates: Partial<IUpdateTeamByTeamIdQueryParams>,
-  teamId: string
-) => {
-  await catchAndLog(() =>
+  teamIds: string | string[]
+): Promise<IUpdateTeamByTeamIdQueryResult[]> => {
```

Comment:
Let's have some fun with typescript!
Declaring the return type can be brittle because if the underlying query changed (like it did here) you have to update it. There's also the possibility that this will return `Promise<null>` if the write to PG fails.

Instead, let's let TS do the work for us by rewriting catchAndLog:

```ts
const catchAndLog = async <T extends (...args: any[]) => any>(riskyFn: T) => {
  try {
    return (await riskyFn()) as ReturnType<T>
  } catch (e) {
    sendToSentry(e, {tags: {migration: 'postgres'}})
  }
  return null
}
```

with that change, you can get rid declaring the return type here & everywhere else :tada:

Reply:
oh that's a good point. my thinking was that explicit is usually better, but i really like the suggestion of going with implicit return types to offload all of that upkeep to typescript.